specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2021-01-25/guestconfiguration.json

```diff
@@ -1317,7 +1317,7 @@
         },
         "resourceType": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
Seems guestconfiguration.json only supports [VM ](https://github.com/Azure/azure-rest-api-specs/blob/08faf3284d23b2e4d22568563281d14ff0956529/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L30) and [HybridComputeMachine](https://github.com/Azure/azure-rest-api-specs/blob/08faf3284d23b2e4d22568563281d14ff0956529/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L371), right? If yes, this property shouldn't be writeable, right? If I am wrong, please correct me. Thanks.